front/components/sparkle/navigation.tsx

```diff
@@ -201,24 +203,45 @@ export const subNavigationBuild = ({
         current: current === "developers",
         subMenuLabel: current === "developers" ? subMenuLabel : undefined,
         subMenu: current === "developers" ? subMenu : undefined,
-      },
+      }
     ],
   });
 
-  if (isDevelopmentOrDustWorkspace(owner)) {
+  const hasDevOrOwnerAccess = isDevelopmentOrDustWorkspace(owner)
```

Comment:
@spolu added labs_transcripts whitelist feature flag, and showing the labs menu only if any of the labs feature is enabled. 
Showing the "Advanced" menu if any of the labs or extract menus are available

Reply:
Let’s go one step further and move extract below the labs page entirely?